editor/plugins/script_editor_plugin.cpp

```diff
@@ -2063,6 +2064,11 @@ void ScriptEditor::_update_script_names() {
 				case SORT_BY_PATH: {
 					sd.sort_key = path;
 				} break;
+				case SORT_BY_MODIFIED: {
+					// XOR with UINT64_MAX results in descending order
```

Comment:
```suggestion
					// XOR with UINT64_MAX results in descending order.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).